ui/src/kapacitor/utils/alertMessageValidation.ts

```diff
@@ -1,24 +1,33 @@
 import _ from 'lodash'
 import {RULE_MESSAGE_TEMPLATE_TEXTS} from 'src/kapacitor/constants'
 
+// the following template variables must also pass validation, see #5492
+// they are not that much common to be part of RULE_MESSAGE_TEMPLATE_TEXTS
+const EXTRA_MESSAGE_VARIABLES = ['.Time.Unix', '.Time.UnixNano']
+
 export const isValidTemplate = (template: string): boolean => {
-  const exactMatch = !!_.find(RULE_MESSAGE_TEMPLATE_TEXTS, t => t === template)
+  if (
+    !!_.find(RULE_MESSAGE_TEMPLATE_TEXTS, t => t === template) ||
+    !!_.find(EXTRA_MESSAGE_VARIABLES, t => t === template)
+  ) {
+    return true
```

Comment:
Seems like `returning true` here could cause a false positive. if an exact match occurs, returning would be fine, but if it is not an exact match (only contains a time variable) it seems that would be an invalid template. maybe?

Reply:
Yes, it is an exact match here. UI only validates the [go template](https://golang.org/pkg/text/template/) with a pre-defined set of variables that are used by the server. And yes, there are more cases, in which UI validation may fail on a good template. This defect only helps a bit as requested, the time/value ratio for making this perfect for all possible cases is really high ... therefore regexp approach was chosen at the beginning and I am continuing to do so.